test/phpunit/UtilsTest.php

```diff
@@ -55,6 +55,11 @@ class UtilsTest extends CommonClassTest
 	 */
 	public function testExecuteCLI()
 	{
+		// Needs ls. Skip test if not running on *nix system.
```

Comment:
May be you can jsut replace the "ls" with "dir" for windows ? So the executCLI is also executed in Windows ?

Reply:
It's more complex than that - the path would be invalid as it contains forward slashes.
What is the default method to convert a path with mixed '\` and '/' to a path compatible with the windows system.  I did not notice a function doing that.